contrib/tools/meta.md/cterms.md

```diff
@@ -397,20 +412,20 @@ Note that we take advantage of the fact that generated constraints will bubble t
 
     op purify : Module Module TermList -> [CTerm] .
     -----------------------------------------------
-    eq purify(M, M', empty)          = empty .
-    eq purify(M, M', (NeTL , NeTL')) = purify(M, M', NeTL) , purify(M, M', NeTL') .
+    eq purify(M, M', empty)      = empty .
+    eq purify(M, M', (T , NeTL)) = purify(M, M', T) , purify(M, M', NeTL) .
 ```
 
 If the term is well-formed in the first module, return it.
 If the top symbol of the term is from the first module, purify the subterms.
 Otherwise, generate an equality constraint at the top and purify with respect to the second module.
 
 ```maude
-    ceq purify(M, M', T)     = T                       if wellFormed(M, T) .
-    ceq purify(M, M', Q[TL]) = Q[purify(M, M', TL)]    if Q inO asTemplate(M) .
-    ceq purify(M, M', Q[TL]) = FV | ((FV ?= T) /\ QFF) if (not Q inO asTemplate(M)) /\ Q inO asTemplate(M')
-                                                       /\ T | QFF := purify(M', M, Q[TL])
-                                                       /\ FV      := joint-variable(M', M, T) .
+    ceq purify(M, M', T)     = T                                    if wellFormed(M, T) .
+    ceq purify(M, M', T)     = FV | (FV ?= T)                       if wellFormed(M', T) /\ FV := joint-variable(M', M, T) .
+     eq purify(M, M', T | EqC) = purify(M, M', T) | EqC             .
```

Comment:
Move this `eq` up with the other `eq` for `purify`, and line up the equal signs. Early equations are fired earlier.

Reply:
Done.